README.md

```diff
@@ -124,6 +124,8 @@ performance.
    </script>
    ```
 
+To customize, use `littlefoot.littlefoot()` instead and pass the settings there.
```

Comment:
If `default` doesn't work, I suggest replacing `littlefoot.default()` in the example above with a comment. Something like:

```
littlefoot.littlefoot() // Pass any littlefoot settings here.
```